docs/smartcard.md

```diff
@@ -41,18 +35,21 @@ A voter card does _not_ use the long value on the card.
 A poll worker card includes the following additional fields in the short value's
 serialized JSON:
 
+- `t` -- `pollworker`
 - `h` -- the base64-encoded SHA256 of the election for which this is a valid
   poll worker card
 
-A poll-worker card does _not_ use the long value on the card.
+A poll worker card does _not_ use the long value on the card.
```

Comment:
nit: the dash is grammatically correct :)

Reply:
Grammar aside… I have yet to see a hyphen used any voting material other than in URLs: https://www.google.com/search?q=%22poll-worker%22+%22poll-worker%22+%22poll-worker%22